indicators/models.py

```diff
@@ -215,6 +215,9 @@ class DisaggregationValue(models.Model):
     disaggregation_label = models.ForeignKey(DisaggregationLabel, on_delete=models.deletion.PROTECT)
     indicator = models.ForeignKey('indicators.Indicator', null=True, blank=True)
     value = models.CharField(max_length=765, blank=True)
+    table_uuid = models.CharField(max_length=765, blank=True)
```

Comment:
Why don't you use UUIDField? It will optimise the storage length and will provide validation.

Reply:
Almost every service we have is using charfield for Foreign Keys uuid. Otherwise we need to convert it for every JSON request. table_uuid is uuid for table in track service.